kolibri/content/models.py

```diff
@@ -3,6 +3,31 @@
 To access it, please use the public APIs in api.py
 
 The ONLY public object is ContentNode
+
+Developer note: If you modify the schema here, it has implications for the content
+import pipeline, including Kolibri Studio where the imported content databases
+are created.
```

Comment:
Currently, the code is copy pasted. The eventual plan is to move everything to abstract base classes so that they can be imported to both Kolibri and Studio and instantiated.

Will add a note about the current state of play.